src/positions/Positions.tsx

```diff
@@ -0,0 +1,147 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { ActivityIndicator, Image, StyleSheet, Text, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import TokenDisplay from 'src/components/TokenDisplay'
+import { positionsSelector, positionsStatusSelector } from 'src/positions/selectors'
+import { BaseToken, Position, Token } from 'src/positions/types'
+import { getFeatureGate } from 'src/statsig'
+import { StatsigFeatureGates } from 'src/statsig/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import variables from 'src/styles/variables'
+import { tokensByAddressSelector } from 'src/tokens/selectors'
+import { Currency } from 'src/utils/currencies'
+
+function getBaseTokens(tokens: Token[]): BaseToken[] {
+  return tokens.flatMap((token) => {
+    if (token.type === 'base-token') {
+      return [token]
+    } else {
+      return getBaseTokens(token.tokens)
+    }
+  })
+}
+
+function PositionItem({ position }: { position: Position }) {
+  const tokens = useSelector(tokensByAddressSelector)
+
+  const baseTokens = getBaseTokens(position.tokens)
+  const baseTokenImages = baseTokens
+    .map((token) => {
+      const tokenInfo = tokens[token.address]
+      return tokenInfo ? tokenInfo.imageUrl : undefined
+    })
+    .filter((image) => image !== undefined)
+
+  const balanceInDecimal =
+    position.type === 'contract-position' ? undefined : new BigNumber(position.balance)
+  const balanceUsd =
+    position.type === 'contract-position'
+      ? new BigNumber(position.balanceUsd)
+      : new BigNumber(position.balance).multipliedBy(position.priceUsd)
+
+  return (
+    <View style={styles.tokenContainer}>
+      <View style={styles.row}>
+        <View style={{ flexDirection: 'row-reverse' }}>
```

Comment:
nit: use the stylesheet instead of inline? 🙈

Reply:
yep 👍